app/proc/telegram.go

```diff
@@ -75,7 +75,7 @@ func (client TelegramClient) Send(channelID string, item feed.Item) (err error)
 func (client TelegramClient) sendText(channelID string, item feed.Item) (*tb.Message, error) {
 	message, err := client.Bot.Send(
 		recipient{chatID: channelID},
-		client.getMessageHTML(item, true),
+		client.getMessageHTML(item, true, false),
```

Comment:
a small suggestion: having two bool params one after another is hard to reason about, especially without IDE help. Adding a small private struct like sendParams and passing it, will make things more readable